### ISSUE TITLE
Pip installs 0.38 but there is a 0.40.dev0 on Pypi

### ISSUE THREAD

**Issue by andy-maier** [webpy/webpy#397]
This installs v0.38 from Pypi:

```
$ pip install web.py
```

However, the latest version on Pypi is 0.40.dev0.

I don't think it is a good idea to release a `.devX` version to Pypi. One reason is that apparently, Pip ignores it.

**Comment by siloam** [comment]
I completely agree with you. We (users) also need a changelog, because we know nothing about new functionalities or fixes.

**Comment by tschertel** [comment]
Webpy is being neglected since Aaron died.
The actual maintainer does not keep this project running as it should. I know I'll be thrown into the fire for saying this.
Webpy is a great framework, but it should have more attention.
I'm not saying I should maintain this repo. I don't have time neither have knowledge to keep an open source project.
But I bet there's a lot of good people who can make this a great project again.

**Comment by anandology** [comment]
@Tomas, I agree that I'm spending enough time on web.py as a maintainer
should. That doesn't mean that I've lost interest in web.py. I still use
web.py heavily and will be happy to consider if you any suggestions to
improve the project.

**Comment by anandology** [comment]
On Fri, Jan 13, 2017 at 11:50 PM, Andreas Maier <notifications@github.com>
wrote:

> This installs v0.38 from Pypi:
>
> $ pip install web.py
>
> However, the latest version on Pypi is 0.40.dev0.
>
> I don't think it is a good idea to release a .devX version to Pypi. One
> reason is that apparently, Pip ignores it.
>

I think that is the whole point. dev versions can be pushed to pypi and
people who install it using the ==0.40.dev0 will be able to install it, but
it won't effect the others. Isn't that a simpler way to share dev versions
with everyone?

**Comment by tschertel** [comment]
@anandology Maybe you should spread the word webpy is looking for a full time maintainer!
Announce in hacker news, and any other news channel available.
You can still decide what will be merged and keep webpy loyal to it's initial purpose.

**Comment by anandology** [comment]
@tschertel I think that is a good idea. I'll start an email thread in web.py mailing list shortly.

**Comment by jzellman** [comment]
Hi @siloam, @tschertel 

Regardless if anyone believes @anandology is a good enough maintainer or is spending enough time, one thing is for certain, he is spending *the most* time maintaining the project. Are there people waiting to be asked to maintain this library or any other one? I feel that if these people had the passion, they'd already be contributing to the project. Maybe a more realistic approach is for users of the project to contribute back things they deem need to be addressed. Rather than point out a defect of the code or documentation, make a pull request addressing the concern. For example, if the lack of a change log is a problem, make a pull request with your best guess for the change log. Harness source control and go through the commit log and build up a text file. Then ask users of the project to contribute to improve the pull request. 

It's sometimes easy to forget that open source projects are a labor of love. @anandology is spending his most crucial asset, time, on the project. I know there are a number of open PRs. And again, these take time to read, follow up, merge etc. Rather than treat the project like a stray dog and drop it off somewhere to find it a new home, perhaps we can work collectively to help @anandology through these PRs and issues. There are a number of areas that don't require intimate knowledge, like documentation, web site fixes, etc that we can help out with. It's hard to pin all this on one guy who likely has a family and a life outside of code. 

It's easy for me to type this all out, but actions speak louder than words. So I will be going through all 69 PRs today and providing feedback/improvements. Next will be the open issues.

With this issue, I think it should be closed with the following: the dev version should be taken off pip, an email should be sent (to the google group?) to test the latest dev release and how to install it from github or whereever. The other discussion of finding a new maintainer should be taken off github issues and on to the mailing list as well. Maybe a new maintainer isn't needed. Maybe a set of core contributors is needed to take some of the work load off @anandology. I think the mailing list is a better place to discuss this. @anandology please reach out to me if I can help with the above 3 things.

@siloam if you open a PR with a change log file, please @ message me and I will contribute to it.

**Comment by gardiner** [comment]
@jzellman Thanks for your constructive and helpful response. I could not agree more. Cheers!

**Comment by tschertel** [comment]
All this was asked in ML,maybe three years ago.
My point is that @anandolog don't have time for keep this project running
as I think it should. I can't say it's his fault. I'll never point fingers
saying he is not enough for the project. He just don't get enough time.
Just it.
Maybe you're right and just give permissions to a small group of
contributors is enough. But one thing I keep saying: webpy need more
attention to keep growing.



--
Tomás Schertel

**Comment by macropin** [comment]
@anandology It's clear that you don't have time for this project. Please delegate access to those that can contribute. Github makes this really easy!

If you don't know who to delegate to then, may I suggest starting with the people who contributed the 70 PR's that are waiting review.

**Comment by suhashpatil** [comment]
+1


Suhas Patil - Linked In <http://www.linkedin.com/in/suhashpatil>
*Khojpal - Connect Locally! *
Facebook <https://www.facebook.com/khojpal/> :: Web
<http://www.khojpal.com/> :: Android
<https://play.google.com/store/apps/details?id=com.affingo.khojpal> :: iOS
<https://itunes.apple.com/us/app/id1035269012>
Press: Economic Times
<http://economictimes.indiatimes.com/small-biz/startups/from-finding-blood-donors-to-renting-sherwani-new-apps-make-local-connection-easier/articleshow/53705557.cms>,
Times of India
<http://timesofindia.indiatimes.com/city/pune/New-to-the-town-These-apps-help-you-find-a-friend/articleshow/52059326.cms>
Mobile: +91 (982) 284-2031
Skype: suhaspatil
